src/test/java/Test/ChangeOrderStatus.java

```diff
@@ -0,0 +1,20 @@
+package Test;
+
+import org.testng.annotations.Test;
+
+public class ChangeOrderStatus extends BaseClass {
```

Comment:
Test name should be start with Test

Ex. TestToVeiftAdminIsAbleToChangeOrderStaus